rocketpy/motors/Tank.py

```diff
@@ -7,15 +7,73 @@
 from abc import ABC, abstractmethod
 
 from rocketpy.Function import Function, funcify_method
-from rocketpy.utilities import except_negative
 
 
 class Tank(ABC):
-    def __init__(self, name, geometry, gas, liquid=0):
+    """Abstract Tank class that defines a tank object for a rocket motor, so
+    that it evaluates useful properties of the tank and its fluids, such as
+    mass, volume, fluid flow rate, center of mass, etc.
+    """
+
+    def __init__(self, name, geometry, flux_time, gas, liquid, discretize=100):
+        """Initialize Tank class.
+
+        Parameters
+        ----------
+        name : str
+            Name of the tank.
+        geometry : TankGeometry
+            Geometry of the tank.
+        flux_time : float, tuple of float, optional
+            Tank flux time.
```

Comment:
Is `flux time` a normally used term? Might be important to explain what it represents/does/is

Reply:
Solved by [f398390](https://github.com/RocketPy-Team/RocketPy/pull/354/commits/f39839020dc1ccf834968faf14de94f9026d9a11).